lib/ReactViews/Notification/NotificationWindow.jsx

```diff
@@ -49,11 +49,16 @@ const NotificationWindow = createReactClass({
       typeof this.props.title === "function"
         ? this.props.title(this.props.viewState)
         : this.props.title ?? "";
-    const message = parseCustomMarkdownToReact(
+
+    let message =
       typeof this.props.message === "function"
         ? this.props.message(this.props.viewState)
-        : this.props.message
-    );
```

Comment:
Previously, this was potentially passing a `ReactNode` into `parseCustomMarkdownToReact()`